app/assets/stylesheets/content/work_packages/timelines/_timelines_header.sass

```diff
@@ -21,7 +24,7 @@
   white-space: nowrap
 
   &.-top-border
-    border-top: 1px solid $timeline--header-border-color
+    outline: 1px solid $timeline--header-border-color
```

Comment:
As discussed, a class `top-border` which adds an outline is a bit ugly. So please check whether we can manage without this change or rename the class and removed the doubled right border.

Reply:
I reverted the last commit and picked up this topic as a new ticket here: https://community.openproject.com/projects/openproject/work_packages/29389/activity